### PR TITLE
docs(all-services): add arc branding and improve documentation

### LINKED ISSUE #1440: docs: revisit and improve documentation of all packages
- [x] docs: catalog root

Packages

- [x] #1442
- [ ] docs: `@sourceloop/core`
- [ ] docs: `@sourceloop/cli`
- [ ] docs: `@sourceloop/feature-toggle`
- [ ] docs: `@sourceloop/search`

Services

- [x] #1443
- [ ] docs: `@sourceloop/authentication-service`
- [ ] docs: `@sourceloop/bpmn-service`
- [ ] docs: `@sourceloop/chat-service`
- [ ] docs: `@sourceloop/feature-toggle-service`
- [ ] docs: `@sourceloop/in-main-service`
- [ ] docs: `@sourceloop/notification-service`
- [ ] docs: `@sourceloop/oidc-service`
- [ ] docs: `@sourceloop/payment-service`
- [ ] docs: `@sourceloop/reporting-service`
- [ ] docs: `@sourceloop/scheduler-service`
- [ ] docs: `@sourceloop/search-service`
- [ ] #1659
- [ ] docs: `@sourceloop/video-conferencing-service`

### PR DESCRIPTION
**Pull request by Surbhi-sharma1**
GH-1440

## Description

add arc branding and improve documentation

Fixes #1440 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: services/authentication-service/README.md [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
+# [@sourceloop/authentication-service](https://github.com/sourcefuse/loopback4-microservice-catalog/tree/master/services/authentication-service)
+
+<p align="left">
+<a href="https://www.npmjs.org/package/@sourceloop/authentication-service">
+<img src="https://img.shields.io/npm/v/@sourceloop/authentication-service.svg" alt="npm version" />
+</a>
+<a href="https://github.com/sourcefuse/loopback4-microservice-catalog/graphs/contributors" target="_blank">
+<img alt="GitHub contributors" src="https://img.shields.io/github/contributors/sourcefuse/loopback4-microservice-catalog">
+</a>
+<a href="https://www.npmjs.com/@sourceloop/audit-service" target="_blank">
```
Comment (yeshamavani) [inline services/authentication-service/README.md:12]: audit?

File: services/chat-service/README.md [hunk 2]
```diff
@@ -1,16 +1,30 @@
...
+# [@sourceloop/chat-service](https://github.com/sourcefuse/loopback4-microservice-catalog/tree/master/services/chat-service)
+
+<p align="left">
+ <a href="https://nodejs.org/en/" target="_blank">
+  <img src="https://img.shields.io/node/v/@sourceloop/chat-service" alt="Node.js version (scoped)" />
+</a>
+<a href="https://github.com/sourcefuse/loopback4-microservice-catalog/graphs/contributors" target="_blank">
+<img alt="GitHub contributors" src="https://img.shields.io/github/contributors/sourcefuse/loopback4-microservice-catalog">
+</a>
+<a href="https://www.npmjs.com/@sourceloop/audit-service" target="_blank">
```
Comment (yeshamavani) [inline services/chat-service/README.md:12]: audit?

File: services/authentication-service/README.md (hunk 1 above)
Comment (Surbhi-sharma1) [reply services/authentication-service/README.md]: changed

File: services/chat-service/README.md (hunk 2 above)
Comment (Surbhi-sharma1) [reply services/chat-service/README.md]: changed

**Pull request merged**